apps/nrm/views/pv-under-age.html

```diff
@@ -1,6 +1,10 @@
 {{<partials-page}}
   {{$page-content}}
-    {{#markdown}}pv-under-age{{/markdown}}
+  <div  tabindex="0">
```

Comment:
extra space here should be deleted.  
```suggestion
  <div tabindex="0">
```